storage/uri.go

```diff
@@ -109,3 +109,15 @@ func Parent(u fyne.URI) (fyne.URI, error) {
 	parent = parent + strings.Join(components[0:len(components)-1], "/") + "/"
 	return NewURI(parent), nil
 }
+
+// Join appends a new path element to a URI, separated by a '/' character.
+func Join(u fyne.URI, component string) fyne.URI {
```

Comment:
I know when discussing 2.0 URI we agreed that Child() would make sense, if there is an intermediate method that is public can it be consistent - something like storage.ChildURI()?

Reply:
I think we already have `storage.Parent()`, so I would be inclined to go with `storage.Child()` to be consistent with that, moving it into the URI interface in 2.0. This would also apply to `storage.Exists()`.

Reply:
Yeah, that sounds like a plan